src/main/java/ua/tc/marketplace/service/CategoryService.java

```diff
@@ -0,0 +1,43 @@
+package ua.tc.marketplace.service;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import ua.tc.marketplace.model.entity.Category;
+import ua.tc.marketplace.repository.CategoryRepository;
+
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    public Page<Category> findAll(Pageable pageable) {
+        return categoryRepository.findAll(pageable);
+    }
+
+    public Optional<Category> findById(Long id) {
+        return categoryRepository.findById(id);
```

Comment:
when you will return the logic in service method, due to @OleksandrProkopenkoDev coment, throw exception in case when nothing found